aws-globalaccelerator-accelerator/inputs/inputs_1_create.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Can we add README,md for accelerator resource as well?

Reply:
Whoops,  missed being part of commit.  Fixed